### DISCUSSION TITLE
Can OpenVDB currently be built for Python 3.10?

### DISCUSSION THREAD

**Discussion by 8bit-euclid** [AcademySoftwareFoundation/openvdb#1599] [Q&A]
I have been tasked with getting OpenVDB to build with Python 3.10. Is the latest version of OpenVDB compatible with Python 3.10. I have so far only found build instructions for OpenVDB supporting up to Python 3.7. 
If this is possible, could someone please advise me on how to do this build?

**Comment by somdats** [comment]
@8bit-euclid , Yes, I built the  openvdb with python 3.10

openvdb installation
 As, I am using a visual studio compiler on a windows PC, I built openvdb with the following steps


git clone https://github.com/Microsoft/vcpkg.git
run x64 visual studio cmd as admin
cd /pathto/vcpkg
./bootstrap-vcpkg.sh
./vcpkg integrate install
vcpkg install pybind11:x64-windows
vcpkg install zlib:x64-windows
vcpkg install blosc:x64-windows
vcpkg install tbb:x64-windows
vcpkg install boost-iostreams:x64-windows
vcpkg install boost-any:x64-windows
vcpkg install boost-algorithm:x64-windows
vcpkg install boost-uuid:x64-windows
vcpkg install boost-interprocess:x64-windows
vcpkg install boost-python:x64-windows  
vckpkg install glfw3:x64-windows
vcpkg install glew:x64-windows

git clone git@github.com:AcademySoftwareFoundation/openvdb.git
cd openvdb
mkdir build
cd build

cmake -DCMAKE_TOOLCHAIN_FILE=D:\vcpkg\scripts\buildsystems\vcpkg.cmake -DVCPKG_TARGET_TRIPLET=x64-windows -A x64 -DOPENVDB_BUILD_VDB_VIEW=ON  -DOPENVDB_BUILD_VDB_TOOL=ON -DOPENVDB_BUILD_VDB_LOD=ON -DOPENVDB_BUILD_PYTHON_MODULE=ON ..

finally
cmake --build . --parallel 4 --config Release --target install
